### DISCUSSION TITLE
Furaffinity rss question

### DISCUSSION THREAD

**Discussion by Destroyer77d** [DIYgod/RSSHub#9242] [Q&A] [answered]
Hi, I've been trying to find the appropriate place to discuss this. I've been trying to use the format listed here: https://docs.rsshub.app/en/social-media.html#fur-affinity  to take furaffinity RSS feeds and add them to my RSS reader. However, because of my lack of experience with this, the part saying: "nsfw, optional - NSFW Mode, do not filter NSFW contents when value set to 1" has confused me. I'm not sure what change I need to make when using their format will allow it to show all of the pictures in the user feeds' 

The full segment on the page I'm trying to figure it out for is: 
User's Gallery
[support browser extension](https://github.com/DIYgod/RSSHub-Radar)
Author: [@TigerCubDen](https://github.com/TigerCubDen)

Example: https://rsshub.app/furaffinity/gallery/flashlioness 

Route: /furaffinity/gallery/:username/:nsfw?

Parameters:
username, required - Username, can find in userpage
nsfw, optional - NSFW Mode, do not filter NSFW contents when value set to 1

I'd appreciate any help. Thanks!

**Comment by TonyRL** [comment, marked as answer]
No filtering: `/furaffinity/gallery/flashlioness/1`
Filter NFSW: `/furaffinity/gallery/flashlioness`

**Reply by Destroyer77d** [reply]
Ah! Thank you thank you thank you thank you!
